src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseService.cs

```diff
@@ -254,15 +257,14 @@ private async Task<List<ContentSection>> TemplateFromRelease(Guid? releaseId)
                 .OnSuccess(async footnotes =>
                 {
                     var subject = await _subjectService.GetAsync(releaseId, subjectTitle);
-                    var dependentDataBlocks = GetDependentDataBlocks(releaseId, subject.Id);
-
-                    var orphanFootnotes = await _subjectService.GetFootnotesOnlyForSubjectAsync(subject.Id);
+                    var dependentDataBlocks = GetDependentDataBlocks(releaseId, subject?.Id ?? Guid.Empty);
```

Comment:
I think it would be clearer if we just take out these two steps if there's no subject id